jdaviz/configs/specviz2d/helper.py

```diff
@@ -238,3 +238,25 @@ def load_trace(self, trace, data_label, show_in_viewer=True):
             self.app.add_data_to_viewer(
                 self._default_spectrum_2d_viewer_reference_name, data_label
             )
+
+    def get_data(self, data_label=None, spectral_subset=None, cls=None):
+        """
+        Returns data with name equal to data_label of type cls with subsets applied from
+        spectral_subset.
+
+        Parameters
+        ----------
+        data_label : str, optional
+            Provide a label to retrieve a specific data set from data_collection.
+        spectral_subset : str, optional
+            Spectral subset applied to data.
+        cls : `~specutils.Spectrum1D`, `~astropy.nddata.CCDData`, optional
+            The type that data will be returned as.
+
+        Returns
+        -------
+        data : cls
+            Data is returned as type cls with subsets applied.
+
+        """
+        return self._get_data(data_label=data_label, spectral_subset=spectral_subset, cls=cls)
```

Comment:
I think mosviz will need to have the same defined so that it has access to `spectral_subset`.

Reply:
Done.